std/utf.d

```diff
@@ -3829,14 +3824,12 @@ template byUTF(C) if (isSomeChar!C)
                         {
                             static if (is(RC == dchar))
                             {
-                                fill = cast(ushort) encode!(Yes.useReplacementDchar)(buf, c);
                                 r.popFront;
+                                dchar dc = c;
                             }
                             else
-                            {
```

Comment:
Personally, I see no problem with having braces on one branch and not on another. If the branch is only one line, then braces are just noise. But the style guide doesn't say anything on the matter.